### PR TITLE
[REVIEW] Update notebooks to use the new Tokenizer

### PR DESCRIPTION
**Pull request by VibhuJawa**
This PR closes https://github.com/rapidsai/clx/issues/418. 

Files to change:
- [x] sequence_classifier.py
- [x] pii_detection_training_example.ipynb
- [x] sequence_classifier.py
- [x] cybert.py
- [x] cybert_example_training.ipynb

### CODE REVIEW

File: python/clx/analytics/cybert.py [hunk 1]
```diff
@@ -114,28 +118,21 @@ def preprocess(self, raw_data_col, stride_len=116, max_seq_len=128):
...
-            self._hashpath,
-            128,
-            116,
-            max_rows_tensor=max_rows_tensor,
-            do_lower=False,
-            do_truncate=False,
+        output = self.tokenizer(
+            raw_data_col,
+            max_length=128,
+            stride=12,
```
Comment (VibhuJawa) [inline python/clx/analytics/cybert.py:124]: We changed stride here to match hugging face. 
  https://github.com/rapidsai/cudf/blob/5b8895dc50a5daf99bafede67617e8ebd39d3af6/python/cudf/cudf/core/subword_tokenizer.py#L207

File: python/clx/analytics/sequence_classifier.py [hunk 2]
```diff
@@ -117,7 +121,7 @@ def evaluate_model(self, test_data, labels, max_seq_len=128, batch_size=32):
         eval_accuracy = 0
         nb_eval_steps = 0
         for df in test_dataloader.get_chunks():
-            b_input_ids, b_input_mask = self._bert_uncased_tokenize(df["text"], max_seq_len)
+            b_input_ids, b_input_mask = self._bert_uncased_tokenize(self._tokenizer, df["text"], max_seq_len)
```
Comment (VibhuJawa) [inline python/clx/analytics/sequence_classifier.py:124]: Unsure where this function is used so was not able to test it.

Comment (VibhuJawa) [reply python/clx/analytics/sequence_classifier.py:124]: Looks like covered under CI.

**Comment by VibhuJawa** [comment]
The CI failure on the PR looks independent to the  PR. In a fresh environment below fails:
```python
(rapids) root@37bcf6ec0c27:/clx/python/clx# pytest tests/test_windows_event_parser.py::test_windows_event_parser
==================================================================================================== test session starts ====================================================================================================
platform linux -- Python 3.7.10, pytest-6.2.4, py-1.10.0, pluggy-0.13.1
rootdir: /clx/python, configfile: pytest.ini
plugins: anyio-3.2.0
collected 1 item                                                                                                                                                                                                            
.....
.....
.....
.....
.....
.....
.....

>       assert parsed_rec["detailed_authentication_information_logon_process"] == "ntlmssp"
E       AssertionError: assert 'ntlmssp ' == 'ntlmssp'
E         - ntlmssp
E         + ntlmssp 
E         ?        +

tests/test_windows_event_parser.py:136: AssertionError

```

**Comment by VibhuJawa** [comment]
Thanks for linking that @efajardo-nv .  The PR is ready for a review then. :-D

**Comment by VibhuJawa** [comment]
@gpucibot  rerun tests

**Comment by efajardo-nv** [comment]
rerun tests

**Review by efajardo-nv — APPROVED** [review]

**Comment by efajardo-nv** [comment]
@gpucibot merge

**Pull request merged**